src/libraries/Common/src/Interop/Interop.TimeZoneInfo.iOS.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Globalization
+    {
+        [LibraryImport(Libraries.GlobalizationNative, EntryPoint = "GlobalizationNative_GetTimeZoneDisplayNameNative", StringMarshalling = StringMarshalling.Utf16)]
+        internal static unsafe partial ResultCode GetTimeZoneDisplayNameNative(
```

Comment:
Do we need partial here?

Reply:
No, copy pasted from https://github.com/dotnet/runtime/blob/1487d46e4ff7731b52af92b1228373a2a66ed6bf/src/libraries/Common/src/Interop/Interop.TimeZoneInfo.cs , but there also looks like it is not needed.

Reply:
Actually all functions in Interop.Globalization are `partial`